### ISSUE TITLE
[APP SUBMITTED]: Missing time zone for network: Sky Atlantic (IT)

### ISSUE THREAD

**Issue by phraymond** [pymedusa/Medusa#2619]
### INFO
**Python Version**: `2.7.13 (default, Dec 22 2016, 09:22:15) [GCC 6.2.1 20160822]`
**Operating System**: `Linux-4.9.19-unRAID-x86_64-with`
**Locale**: `None`
**Branch**: [master](../tree/master)
**Database**: `44.4`
**Commit**: pymedusa/Medusa@a0116154f262786d6f5274459cb8f9cdb23012cc
**Link to Log**: https://gist.github.com/361061576b2629333385018825ef624b
### ERROR
<pre>
2017-04-20 08:57:34 ERROR    Thread-20 :: [a011615] Missing time zone for network: Sky Atlantic (IT)
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by fernandog** [comment]
@phraymond Is this your first time in Medusa ? can you confirm you got a warning log like this?

 `Updating network timezones failed, this can happen from time to time`

**Comment by fernandog** [comment]
@phraymond or this one: `Unable to build the network dictionary. Aborting update. Error`

**Comment by phraymond** [comment]
@fernandog Odd, I didn't get any notifications on this, sorry. Fixed. To answer the question, yes, first time in Medusa (although this happened in Sickrage about once a week, one of the reasons I moved to Medusa). As for the error messages, not sure. I ended up getting frustrated with the ton of errors when I tried to re-use my Sickrage db, so I blew the whole thing away and started from scratch, copy/pasting my settings from the one to the other. It's running through my shows now, adding them to the DB, so I'll have to wait for that to finish. May be awhile, as I've got 38,070 episodes across 1,448 shows. The log shows it's processing but the main page is throwing this up:

Mako Error
A mako error has occured.
If this happened during an update a simple page refresh may be the solution.
Mako errors that happen during updates may be a one time error if there were significant ui changes.

I figure I'll wait for the logs to indicate that the show processing has finished before I worry about that error. Thanks.
